unit_tests/src/test_ticket_administration.cpp

```diff
@@ -0,0 +1,365 @@
+#include <catch2/catch.hpp>
+
+#include "irods/connection_pool.hpp"
+#include "irods/rodsClient.h"
+#include "irods/ticket_administration.hpp"
+#include "irods/query_builder.hpp"
+#include "irods/user_administration.hpp"
+
+#include <fmt/format.h>
+
+#include <string>
+#include <iostream>
+#include <stdio.h>
+#include <stdexcept>
+
+TEST_CASE("ticket administration")
+{
+    namespace tic = irods::experimental::administration::ticket;
+    std::string read_ticket_name = "testTicket";
+    std::string write_ticket_name = "testTicket2";
+    std::string read_string = "read";
+    std::string read_coll_string = "/tempZone/home/rods";
+    std::string write_string = "write";
+    std::string write_coll_string = "/tempZone";
+
+    irods::experimental::query_builder builder;
+
+    load_client_api_plugins();
+    auto conn_pool = irods::make_connection_pool();
+    auto conn = conn_pool->get_connection();
+
+    std::string set_ticket_name = "setTicket";
+    std::string set_ticket_collection = "/tempZone/home";
+    std::string user_name = "testUser";
+    std::string group_name = "testGroup";
+
+    int set_test_number = 19;
+    std::string set_test_number_string = std::to_string(set_test_number);
+    std::string remove_test_number_string = std::to_string(0);
```

Comment:
Let's replace the use of this with `"0"`. No need to compute anything and no need for a variable.